docs/compliance/dist/system-security-plans/ato/cp-10.2.md

```diff
@@ -26,6 +23,6 @@ ______________________________________________________________________
 
 ## What is the solution and how is it implemented?
 
-Add control implementation description here for control cp-10.2
+In accordance with application owner SLAs established with cloud.gov. 
```

Comment:
This feels potentially insufficient because it explains how the solution is implemented ("In accordance with application owner SLAs established with cloud.gov.") but not what the solution is.